cmd/monaco/deploy/command.go

```diff
@@ -56,11 +58,19 @@ func GetDeployCommand(fs afero.Fs) (deployCmd *cobra.Command) {
 				return err
 			}
 
-			return deployConfigs(ctx, fs, manifestName, groups, environment, project, continueOnError, dryRun)
+			maxConcurrentDeployments := environment.GetEnvValueIntLog(environment.ConcurrentDeploymentsEnvKey)
```

Comment:
Is this really necessary to have in the deploy here? The command should IMO not be responsible to set the deployment concurrency - it parses the argument and calls the correct code. I think this should move further down

Reply:
I thought when it comes to env variable handling it should be rather on top. I'll move it to the previous place then :+1: